src/Symfony/Component/Console/Input/InputOption.php

```diff
@@ -146,6 +150,16 @@ public function isArray()
         return self::VALUE_IS_ARRAY === (self::VALUE_IS_ARRAY & $this->mode);
     }
 
+    /**
+     * Returns true if the option is negatable.
+     *
+     * @return bool true if mode is self::VALUE_NEGATABLE, false otherwise
+     */
+    public function isNegatable()
```

Comment:
`: bool`?

And I would remove the whole docblock that does not seem to be needed.